### DISCUSSION TITLE
Transfere open pull request from hashicorp/terraform to opentofu/opentofu

### DISCUSSION THREAD

**Discussion by l-with** [opentofu/opentofu#714] [Q&A] [answered]
Perhaps a stupid question:

I have an open pull request for hashicorp/terraform and signed the Hashicorp CLA.

Is it legal to transfere the pull request to opentofu/opentofu?

**Comment by cube2222** [comment, marked as answer]
Hey! Definitely not a stupid question @l-with, and apologies for the delayed response.

To cite their CLA (emphasis mine):
> Grant of Copyright License. Subject to the terms and conditions of this Agreement, You hereby grant to HashiCorp and to recipients of software distributed by HashiCorp a perpetual, worldwide, **_non-exclusive_**, no-charge, royalty-free, irrevocable copyright **_license_** to reproduce, prepare derivative works of, publicly display, publicly perform, sublicense, and distribute Your Contributions and such derivative works.

It being a non-exclusive license means you can resubmit the code to OpenTofu as well - and that goes for both open and merged pull requests that you're the author of.
